### PR TITLE
Creatkeebs

### PR DESCRIPTION
**Pull request by Timliuzhaolu**
## Description

Add thera from creatkeebs.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/creatkeebs/thera/config.h
Comment (drashna) [inline keyboards/creatkeebs/thera/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/creatkeebs/thera/config.h
Comment (drashna) [inline keyboards/creatkeebs/thera/config.h:2]:
```suggestion
```

File: keyboards/creatkeebs/thera/config.h
Comment (drashna) [inline keyboards/creatkeebs/thera/config.h:47]:
```suggestion
```

File: keyboards/creatkeebs/thera/config.h
Comment (drashna) [inline keyboards/creatkeebs/thera/config.h:50]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/creatkeebs/thera/info.json
Comment (drashna) [inline keyboards/creatkeebs/thera/info.json:1]: This is a via json, and as such does not belong in this repo.

File: keyboards/creatkeebs/thera/keymaps/default/keymap.c
Comment (drashna) [inline keyboards/creatkeebs/thera/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/creatkeebs/thera/keymaps/default/keymap.c
Comment (drashna) [inline keyboards/creatkeebs/thera/keymaps/default/keymap.c:67]:
```suggestion
```

File: keyboards/creatkeebs/thera/keymaps/default/keymap.c
Comment (drashna) [inline keyboards/creatkeebs/thera/keymaps/default/keymap.c:116]:
```suggestion
```

File: keyboards/creatkeebs/thera/readme.md
Comment (drashna) [inline keyboards/creatkeebs/thera/readme.md:18]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/creatkeebs/thera/rules.mk
Comment (drashna) [inline keyboards/creatkeebs/thera/rules.mk:57]:
```suggestion
# Bootloader selection
BOOTLOADER = atmel-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

Also, via should not be enabled at the keyboard level.

File: keyboards/creatkeebs/thera/thera.c
Comment (drashna) [inline keyboards/creatkeebs/thera/thera.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/creatkeebs/thera/thera.h
Comment (drashna) [inline keyboards/creatkeebs/thera/thera.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/creatkeebs/thera/thera.h
Comment (drashna) [inline keyboards/creatkeebs/thera/thera.h:3]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/creatkeebs/thera/info.json
Comment (Timliuzhaolu) [reply keyboards/creatkeebs/thera/info.json:1]: Sorry about it, I have changed it now!

File: keyboards/creatkeebs/thera/keymaps/default/keymap.c
Comment (drashna) [inline keyboards/creatkeebs/thera/keymaps/default/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/creatkeebs/thera/keymaps/default/keymap.c
Comment (Timliuzhaolu) [reply keyboards/creatkeebs/thera/keymaps/default/keymap.c:17]: Yeah, it has been added now. Thanks for pointing it out!

**Review by drashna — APPROVED** [review]

File: keyboards/creatkeebs/thera/readme.md
Comment (fauxpark) [inline keyboards/creatkeebs/thera/readme.md:4]:
```suggestion
![thera](https://i.imgur.com/7w7ARcPh.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by Timliuzhaolu** [comment]
Hi both, is there anything else I have to do before merging? Please let me know. Thanks for all the helps and support! @drashna @fauxpark

**Review by fauxpark — APPROVED** [review]

**Comment by Timliuzhaolu** [comment]
Hi Ryan, will this be merged since I have got two approvals now? Thanks! @fauxpark

**Comment by drashna** [comment]
Thanks!

**Pull request merged**
